zio-http/shared/src/main/scala/zio/http/internal/FormState.scala

```diff
@@ -87,6 +106,7 @@ private[http] object FormState {
         if (!lastByte.isEmpty) {
           if (isBufferEmpty) isBufferEmpty = false
           buffer += lastByte.get
+          bufferSize = bufferSize + 1
```

Comment:
Sorry for the nitpick:

```suggestion
          bufferSize += 1
```

Reply:
Thanks.